### PR TITLE
Add elsed

### PR DESCRIPTION
**Pull request by RaphaelKreft**
implement elsed and add demo conf

### CODE REVIEW

**Comment by RaphaelKreft** [comment]
I will wait for testing with line_matcher once we merged line detection branch

**Review by FelixMoeller3 — APPROVED** [review]

**Comment by FelixMoeller3** [comment]
@RaphaelKreft did you test that this works? And do you have benchmarks for it?

**Pull request merged**
